src/Component/Configuration/Loader/BabelLoader.php

```diff
@@ -0,0 +1,44 @@
+<?php
+namespace Hostnet\Component\Webpack\Configuration\Loader;
+
+use Hostnet\Component\Webpack\Configuration\CodeBlock;
+use Hostnet\Component\Webpack\Configuration\ConfigExtensionInterface;
+use Symfony\Component\Config\Definition\Builder\NodeBuilder;
+
+final class BabelLoader implements LoaderInterface, ConfigExtensionInterface
+{
+    /**
+     * @var array
+     */
+    private $config;
+
+    /**
+     * @param array $config
+     */
+    public function __construct(array $config = [])
+    {
+        $this->config = $config['loaders']['babel'];

```

Comment:
Considering you allow an empty array, maybe you should check if it really exists.